pantheon-decoupled-example.php

```diff
@@ -71,10 +73,9 @@ function pantheon_decoupled_example_menu() {
 function pantheon_decoupled_example_activate() {
 	if ( !get_transient('pantheon_decoupled_example_created') ) {
 		pantheon_decoupled_example_create_post();
+	}
```

Comment:
For some reason menu was not getting created on the first run.
So separated the function